pkgs/development/compilers/llvm/11/default.nix

```diff
@@ -72,12 +72,27 @@ let
       inherit clang-tools-extra_src llvm_meta;
     };
 
+    libllvm-polly = callPackage ./llvm {
+      inherit llvm_meta;
+      enablePolly = true;
+    };
+
+    llvm-polly = tools.libllvm-polly.lib // { outputUnspecified = true; };
+
     clang-unwrapped = tools.libclang.out // { outputUnspecified = true; };
+
+    clang-polly-unwrapped = callPackage ./clang {
+      inherit llvm_meta;
+      inherit clang-tools-extra_src;
+      libllvm = tools.libllvm-polly;
+      enablePolly = true;
+    };
+
     # disabled until recommonmark supports sphinx 3
```

Comment:
Comment no longer matches contents. Looks like upstream no longer requires recommonmark for building manpages ([D83161](https://reviews.llvm.org/D83161)), so we should remove it from `llvm/default.nix`. `llvm-manpages` builds for me :+1: